File: sprint8/By lessons/lesson2code1/lesson2code1.cpp
bool compare(const std::string& first, const std::string& second) {
    if (first.length() != second.length()) {
        return false;
    }

    size_t length = first.length();
    for (size_t i = 0; i < length; ++i) {
        if (first[i] != second[i]) {
            return false;
        }
    }

    return true;
}

File: sprint8/By lessons/lesson2code1/lesson2code1.swift
func compare(first: [Character], second: [Character]) -> Bool {
    if first.count != second.count {
        return false
    }

    let length = first.count
    for i in 0..<length {
        if first[i] != second[i] {
            return false
        }
    }

    return true
}
